Calling/ClientApp/src/components/ChatScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import { Stack } from '@fluentui/react';
+import React, { useEffect, useState } from 'react';
+
+import ChatArea from '../containers/ChatArea';
+import ChatHeader from '../containers/ChatHeader';
+import SidePanel from '../containers/SidePanel';
+import {
+  chatScreenBottomContainerStyle,
+  chatScreenContainerStyle,
+} from './styles/ChatScreen.styles';
+import { SidePanelTypes } from './SidePanel';
+
+interface ChatScreenProps {
+  userId: string;
+  threadMembersError: boolean;
+  endChatHandler(): void;
+  errorHandler(): void;
+  getThreadMembers(): void;
+  getThread(): void;
+  getMessages(): void;
+}
+
+export default (props: ChatScreenProps): JSX.Element => {
+  // People pane will be visible when a chat is joined if the window width is greater than 600
```

Comment:
nit: In chat screen, People pane is opening by default, we should hide it on load.

Reply:
Created a separate task to track this suggestion.